fortitude/src/snapshots/fortitude__format__test__use-operator-and-assignment.f90.snap

```diff
@@ -0,0 +1,8 @@
+---
+source: fortitude/src/format.rs
+expression: string
+---
+module test
+  useother, only:operator(), assignment( = )public :: assignment( = )
+  privateoperator()
+end module
```

Comment:
Looks like there's something strange going on in this one. Maybe `public_statement` and `private_statement` need an `@append_space`, and `use_statement` needs an `@append_hardline`?